### ISSUE TITLE
Quick Rest Reshuffle - Proposed Limitations

### ISSUE THREAD

**Issue by LockerM** [DeckofAdventures/TheGame#156]
### Current System

Currently Quick Rests take 30 minutes and you can use Rest Cards and reshuffle your deck on a Quick Rest. Going from 2 cards left in your deck back to 52 on a quick rest feels like someone chugging a huge stamina potion in 30 minutes rather than a natural recovery of stamina from taking a breather. 

### Limitations to add/consider

1. Quick Rests can only be taken twice per Full Rest. 

2. During a Quick Rest, you can reshuffle up to 26 + VIT Mod cards. 

3. Expending Rest Cards happens only after the rest is successful, meaning they're drawn after the reshuffle. 

*- This means if a Dealer has 10 cards left they reshuffle 26, getting up to 36, then expend 4 rest cards, dropping them to 32. They're still over half their deck, but not feeling fully refreshed as they would be if they ended at 52 and then used 4 cards, dropping them to 48 cards.*

**Comment by CBroz1** [comment]
VIT is between 0 and 6. +VIT doesn't make much a difference, so I might do 2*VIT

But, from accessibility/player perspective, do you want to be counting out 26 cards? 10 seems like a more reasonable number to count table

**Comment by D-D-2** [comment]
The MtG nerd in me says getting 1-6 additional cards back is huge, but yes I also understand that it wouldn't feel like a lot to the average player. 

Counting out 26-32 cards sounds like it could be slow, but I don't think making it an easy number like 10 or 20 solves that counting issue.

**Comment by D-D-2** [comment]
@CBroz1 @LockerM Want to re-open discussion on this issue. After our latest playtest, I'd like to drop the quick rest card reshuffle to 10 cards. To counterbalance this slash, I'd suggest 5*VIT as the additional cards players can reshuffle. I think there is enough going on during a quick rest as it is that the additional time counting cards more closely would be nominal.

**Comment by CBroz1** [comment]
I like the idea of linking it to vitality

Currently rest cards can be
- hp
- pp
- a check to repair armor ( DR 7 - attempted gained AP)

That felt pretty complicated in the moment and I think counting cards will add to it. If we implement the above, I would think about removing nonmagical armor from low levels of play

**Comment by LockerM** [comment]
After discussion with @D-D-2, I think we can limit Quick Rests to 10+(VIT*2) rather than *5. 

So a player with a +2 VIT modifier will redraw 14 cards.

Proposed written in as: "When completing a Quick Rest, shuffle your discard pile. Draw the top 10 (+VIT*2) cards back into your deck. Reshuffle your deck." 

^proposed can be modified as needed

**Comment by CBroz1** [comment]
> "When completing a Quick Rest, shuffle your discard pile. Draw the top 10 (+VIT*2) cards back into your deck. Reshuffle your deck."

My only reservation here is *top*. Feels bad to know this won't include the Fate Card I discarded earlier. But also, shuffling before the draw and shuffling again seems a bit of a hassle.

**Comment by LockerM** [comment]
Returning to the point about rest cards and armor points. 

Armor points from shields do not need to be repaired via rest cards. These can be repaired and the AP they provide are replenished by Major Actions in combat. Otherwise a shield will always be at max AP when readied. 

Armor that has AP on it is rare and hard to find. Only higher levels of play will find armor like that readily. At that stage in the game, I propose we make it a feature/function of Tools that allow you to repair the AP from armor during a rest. 

You must expend 1 rest card per AP during a rest to attempt to repair your armor. Make a DR X Craft Check during your quick rest and draw using your rest card pile. 

^this would eliminate the complexity of repairing AP during a quick rest so the only variables are HP and PP.

**Comment by LockerM** [comment]
Proposed refinement to rest mechanic:

Everyone now has a target card on them at all times. So when a quick rest is initiated, everyone is aiming their "Rest Checks" against that TC. 

Proposal 1: a rest is always a DR 3, no modifier. You are restoring 1 HP or PP no matter what, but a success gives you an additional HP or PP back. A Major Success will have an additional benefit (below) 

Proposal 2: any major successes during a rest are reserved as Fate Cards. 

Proposal 3: we reduce the amount of rest cards down to make the use of them more resource intensive but also have higher benefits (55% chance of each providing 2 instead of 1). 


*Additional Note*
We can introduce powers that add complexity to the rest mechanic but for the core DofA system keep it very straightforward and leave complexity for additional rules and systems that can get implemented on top of it.

**Comment by D-D-2** [comment]
I like this as a way to streamline the quick rest mechanic while also reducing its overall power. What would the total amount of rest cards become after the reduction?

**Comment by LockerM** [comment]
Good point - maybe this is where we pull Vitality in as the modifier? 

You get **1 + Level + VIT Modifier** Rest Cards, replenished on a Full Rest.

So a level 2 Martial with +1 VIT gets 4 Rest Cards, a level 4 Defender with +2 VIT gets 7 Rest Cards. 

This additional incentivizes the gain of VIT as it increases your HP and ability to rest/replenish, and gives players 2 rest cards by default at character creation so they can experience the Rest Mechanic without it being overly complex or high counting.

**Comment by D-D-2** [comment]
That works. Keeps it relatively simple and also ties in VIT still. For clarification: Quick Rests are still allowing for reshuffling 10+(VIT*2) from the top of the deck with these changes, yes? 

Also, what's the order of operations? Can players do their rest cards and then shuffle the top 10+X, or must they shuffle first and then do their rest cards? I can see players doing this strategically to get an Ace back from their discard and then have a chance to hit it with their rest card. Or, hit an Ace on a rest card, use it, and shuffle it back. This isn't necessarily a problem, just something to be aware of when making the decision.

**Comment by LockerM** [comment]
Reply to @D-D-2's latest question: In my conception of the next version of the quick rest system to test would be **no reshuffle on quick rest**. 

This streamlined version would make a quick rest only about drawing rest cards. These can be used to a) regain HP or b) regain PP. No AP regen or reshuffle in this proposed streamlined version for quick rests. 

Outcome I expect from this to test in beta is making stamina feel like a more valuable resource, and the decision to draw rest cards to replenish resources will also drain stamina. The additional piece I would recommend we add simultaneous to removing the reshuffle from quick rests is making the first instance of Fatigue less punishing than a universal lower hand on all checks (TBD in a separate issue).

**Comment by CBroz1** [comment]
> Current: Rest Cards = HP + PP
> Proposed: Rest Cards = 1 + Level + VIT Modifier Rest Cards

Why make this change? In the former case, it is still tied to HP and VIT indirectly through HP (4+level+VIT). If we want to reduce the total, can we just set to HP?
